alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -926,4 +929,9 @@ public static function getPreferredStorageProfileId()
 	{
 		return self::$preferredStorageId;
 	}
+
+	public static function getFallbackStorageProfileId()
```

Comment:
use this function in all places in this file, instead of going to the self::$fall..

inside this function we handle the case where the call did not contained the fallbackSt